internal/xds/rbac/rbac_engine_test.go

```diff
@@ -44,110 +55,189 @@ type addr struct {
 func (addr) Network() string   { return "" }
 func (a *addr) String() string { return a.ipAddress }
 
-// TestRBACEngineConstruction tests the construction of the RBAC Engine. Due to
+// TestChainEngineConstruction tests the construction of the ChainEngine. Due to
 // some types of RBAC configuration being logically wrong and returning an error
 // rather than successfully constructing the RBAC Engine, this test tests both
 // RBAC Configurations deemed successful and also RBAC Configurations that will
 // raise errors.
-func (s) TestRBACEngineConstruction(t *testing.T) {
+func (s) TestChainEngineConstruction(t *testing.T) {
 	tests := []struct {
-		name       string
-		rbacConfig *v3rbacpb.RBAC
-		wantErr    bool
+		name        string
+		rbacConfigs []*v3rbacpb.RBAC
```

Comment:
s/rbacConfigs/policies since that is how they are referred to in `NewChainEngine`